packages/assets-controllers/src/NftController.ts

```diff
@@ -222,41 +218,235 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-type AllowedActions =
+export type AllowedActions =
   | AddApprovalRequest
   | NetworkControllerGetNetworkClientByIdAction;
 
+export type AllowEdEvents =
+  | PreferencesControllerStateChangeEvent
+  | NetworkControllerNetworkDidChangeEvent;
+
+export type NftControllerStateChangeEvent = ControllerStateChangeEvent<
+  typeof controllerName,
+  NftControllerState
+>;
+
+export type NFtControllerEvents = NftControllerStateChangeEvent;
```

Comment:
Typo. All other instances of `NFtControllerEvents` need to be fixed as well.

```suggestion
export type NftControllerEvents = NftControllerStateChangeEvent;
```